lib/src/utils/w3c_header_utils.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:instabug_flutter/src/models/generated_w3c_header.dart';
+import 'package:instabug_flutter/src/models/trace_partial_id.dart';
+
+class W3CHeaderUtils {
```

Comment:
I feel like we should make this a singleton as well instead of all static methods or extract them into separate functions and inject things that we need to mock as parameters. What do you think?